commit b80cf0f5de7720f71189f594125f986193c12f81
Author: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date:   Tue Jul 12 09:36:25 2022 +0200

    Revert "hwloc: bring back the configure error when external hwloc is >= 2.0"
    
    This reverts commit 9eb37be46c75662eb4f6c945144121e3e55360be.

diff --git opal/mca/hwloc/external/configure.m4 opal/mca/hwloc/external/configure.m4
index d951c9a92f1b..411d8ad1c1f2 100644
--- opal/mca/hwloc/external/configure.m4
+++ opal/mca/hwloc/external/configure.m4
@@ -184,21 +184,7 @@ AC_DEFUN([MCA_opal_hwloc_external_CONFIG],[
                [AC_MSG_RESULT([yes])],
                [AC_MSG_RESULT([no])
                 AC_MSG_ERROR([Cannot continue])])
-           AC_MSG_CHECKING([if external hwloc version is lower than 2.0])
-           AS_IF([test "$opal_hwloc_dir" != ""],
-                 [opal_hwloc_external_CFLAGS_save=$CFLAGS
-                  CFLAGS="-I$opal_hwloc_dir/include $opal_hwloc_external_CFLAGS_save"])
-           AC_COMPILE_IFELSE(
-               [AC_LANG_PROGRAM([[#include <hwloc.h>]],
-                   [[
-#if HWLOC_API_VERSION >= 0x00020000
-#error "hwloc API version is greater or equal than 0x00020000"
-#endif
-                   ]])],
-               [AC_MSG_RESULT([yes])],
-               [AC_MSG_RESULT([no])
-                AC_MSG_ERROR([OMPI does not currently support hwloc v2 API
-Cannot continue])])
+
            AS_IF([test "$opal_hwloc_dir" != ""],
                  [CFLAGS=$opal_hwloc_external_CFLAGS_save])
 
